code/game/objects/random/random.dm

```diff
@@ -1988,7 +1988,8 @@
 		/obj/item/pizzabox/vegetable,
 		/obj/item/pizzabox/mushroom,
 		/obj/item/pizzabox/meat,
-		/obj/item/pizzabox/pineapple
+		/obj/item/pizzabox/pineapple,
```

Comment:
GET IT OUT

Reply:
could i just get a little macaroni and gravy?